client/src/components/NavBar/index.tsx

```diff
@@ -132,6 +132,7 @@ const Header = (props: HeaderProps) => {
         <Right>
           <LinkWrapper>
             <Popover2
+              hasBackdrop
```

Comment:
adding backdrop creates a reliable way to detect click outside of the popover element